schemas/context/marketo-lead.schema.json

```diff
@@ -0,0 +1,75 @@
+{
+  "meta:license": [
+    "Copyright 2020 Adobe Systems Incorporated. All rights reserved.",
+    "This work is licensed under a Creative Commons Attribution 4.0 International (CC BY 4.0) license",
+    "you may not use this file except in compliance with the License. You may obtain a copy",
+    "of the License at https://creativecommons.org/licenses/by/4.0/"
+  ],
+  "$id": "https://ns.adobe.com/xdm/context/marketo-lead",
+  "$schema": "http://json-schema.org/draft-06/schema#",
+  "title": "Marketo Lead Schema",
+  "type": "object",
+  "description": "Marketo Lead schema. Includes other Profile mixins like person/work details, opt in/out etc.",
+  "meta:extensible": true,
+  "meta:abstract": true,
+  "meta:intendedToExtend": ["https://ns.adobe.com/xdm/context/profile"],
+  "meta:extends": [
+    "https://ns.adobe.com/xdm/context/person",
+	"https://ns.adobe.com/xdm/context/profile-work-details",
+    "https://ns.adobe.com/xdm/context/lead",
+	"https://ns.adobe.com/xdm/context/optinout",
+	"https://ns.adobe.com/xdm/common/essam"
+  ],
+  "definitions": {
+    "marketo-lead": {
+      "properties": {
+        "marketo:leadId": {
+          "title": "Marketo Lead ID",
+          "description": "Marketo lead id.",
+		  "type": "string"
+        },
+        "marketo:mktPersonId": {
```

Comment:
We should consider using the full `marketo` here `mkt` is not descriptive enough. Given that it's in the `marketo` namespace we could maybe eliminate it altogether.

Reply:
Resolved in the new PR 
https://github.com/adobe/xdm/pull/854